### ISSUE TITLE
Wings: Roundtrip of single value attribute become an array after saving

### ISSUE THREAD

**Issue by elrayle** [samvera/hyrax#4556]
### Descriptive summary

In a valkyrie resource, saving the resource converts single value attributes from String to Array.

### Rationale

The roundtrip conversion from resource to AF object and back to resource should not change the value of an attribute.

### Expected behavior

```
resource = MyResource.new(single_value_attr: "one")
resource. single_value_attr           # "one"
updated_resource = Hyrax.persister.save(resource: resource)
updated_resource. single_value_attr   # "one"
```

NOTE: single_value_attr is "one" before and after the save

### Actual behavior

```
resource = MyResource.new(single_value_attr: "one")
resource. single_value_attr           # "one"
updated_resource = Hyrax.persister.save(resource: resource)
updated_resource. single_value_attr   # ["one"]
```

NOTE: single_value_attr is "one" before and an Array ["one"] after the save

### Steps to reproduce the behavior

TBD - creating a test to demonstrate the problem

**Comment by tpendragon** [comment]
@elrayle This happens when `single_value_attr` is defined as a `Valkyrie::Types::String`?

**Comment by no-reply** [comment]
at first glance, i think this is expected `Wings` behavior.

`ActiveFedora`/Fedroa still treats these attributes as multi-valued RDF properties, and Wings doesn't try to cast back to a single value, since that risks silent data loss.

**Comment by elrayle** [comment]
@tpendragon It happens for all single value types for the reason described by @no-reply.

**Comment by elrayle** [comment]
How Valkyrie handles this [@tpendragon in Slack]...

>It’s just if there’s only one value it picks off the first one in the conversion from Fedora->Resource, and then if it’s defined as a Valkyrie::Set the set type conversion takes care of making it an array again.

>Valkyrie’s conversion logic for Fedora is mad complicated, but https://github.com/samvera/valkyrie/blob/master/lib/valkyrie/persistence/fedora/persister/orm_converter.rb#L524-L537 is the hunk of code for this.

>Postgres (and everything else) uses this: https://github.com/samvera/valkyrie/blob/master/lib/valkyrie/persistence/shared/json_value_mapper.rb#L107-L130

**Comment by elrayle** [comment]
Closing... Addressed by PR #4607 - wings: handle single value fields for valkyrie native models
